src/client/src/components/App.js

```diff
@@ -1,10 +1,32 @@
-import React, { Component } from 'react'; 
- 
+import React, { Component } from "react";
+import Sidebar from "../components/abstract/sidebar";
+import Container from "../components/abstract/layout/Container";
+import ViewParticipants from "../components/pages/viewParticipants";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
```

Comment:
when you install a new package like `react-router-dom` you should push your edited **'dependencies'** in package.json file